apps/spruce/src/pages/projectSettings/tabs/GithubCommitQueueTab/getFormSchema.tsx

```diff
@@ -602,5 +607,7 @@ const GitHubChecksAliasesDescription = (
     </StyledLink>{" "}
     in this project&rsquo;s config YAML instead if Version Control is enabled
     and no aliases are defined on the project or repo page.
+    {projectType === ProjectType.Repo &&
```

Comment:
Confirmed that this worked (ideally, would put a line break before "If enabled" but wasn't working for me).
<img width="715" alt="image" src="https://github.com/user-attachments/assets/86138a88-3281-4ba4-93e0-1bb46147f0c0" />